crypto/s2n_rsa_signing.c

```diff
@@ -92,6 +92,16 @@ int s2n_rsa_pkcs1v15_verify(const struct s2n_pkey *pub, struct s2n_hash_state *d
     return 0;
 }
 
+/* this function returns whether PSS signing is supported */
+int s2n_is_rsa_pss_supported()
+{
+#if RSA_PSS_SUPPORTED
```

Comment:
Looking at this, can we just return RSA_PSS_SUPPORTED?

Reply:
yup, nicer